### PR TITLE
[ADD] last_build: add an orphan page informing on the last build date

### PR DESCRIPTION
**Pull request by AntoineVDV**

### CODE REVIEW

File: content/last_build.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+:orphan:
+:nosearch:
+
+Last build date
+===============
+
+The last time the documentation was build in version |version| was on |today|.
```
Comment (jcs-odoo) [inline content/last_build.rst:7]:
```suggestion
The last time the documentation was built in version |version| was on |today|.
```

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
Wouldn't there be a way to change last_build.rst to last.rst and include something like this:
the last update was on [commit merge date] with [commit].
?
@AntoineVDV

**Review by jcs-odoo — APPROVED** [review]

**Comment by AntoineVDV** [comment]
> Wouldn't there be a way to change last_build.rst to last.rst

Last what? The page is not indexed so I'd rather have an obvious file name.

> and include something like this: the last update was on [commit merge date] with [commit].

Nope, not without an extension.

**Comment by jcs-odoo** [comment]
@AntoineVDV okei alor

**Comment by AntoineVDV** [comment]
@robodoo r+

**Pull request closed without merging**